xtuner/v1/model/compose/intern_s1/modeling_vision.py

```diff
@@ -102,7 +102,7 @@ def forward(
                                    dtype=torch.int32,
                                    device=hidden_states.device)
 
-        attn_output: torch.Tensor = self.attn_impl_func(  # type: ignore
+        attn_output, extra_info = self.attn_impl_func(  # type: ignore
```

Comment:
It's not a good idea to change the attention function signature like this. Instead, we should define an `AttnOutput` type (using TypedDict or namedtuple) to represent the attention result."